src/picongpu/include/traits/attribute/GetChargeState.hpp

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright 2014 Rene Widera
```

Comment:
oh dare you!
(I thought you copied 90% of the file, so keeping the old author is correct - just add yourself)